src/main/java/games/strategy/triplea/oddsCalculator/ta/OddsCalculatorPanel.java

```diff
@@ -855,13 +856,17 @@ void init(final PlayerID id, final List<Unit> units, final boolean land) {
       Collections.sort(categories, Comparator.comparing(UnitCategory::getType, (ut1, ut2) -> {
         final UnitAttachment u1 = UnitAttachment.get(ut1);
         final UnitAttachment u2 = UnitAttachment.get(ut2);
-        // for land, we want land, air, aa gun, then bombarding
+        // For land battles, sort by land, air, can't combat move (AA), bombarding
```

Comment:
@ron-murhammer Can you please explain why this code (before or after the changes in this PR) sorts Air before Can't Combat Move?  It looks like the Air sort code appears _after_ the Can't Combat Move sort code, but the images in the forum post obviously show otherwise.

Reply:
This is just the order that units appear in the battle calc window. Goal is to have "can't combat move" units appear last for the defender. The sort code happens in the inverse order. Essentially pushing naval units to the bottom then "can't combat move" units then air units.

Reply:
It must be the inverse ordering that's making it hard for me to see how the comment lines up with the code.